internal/connection/cluster.go

```diff
@@ -132,42 +106,10 @@ func NewParams(config *hazelcast.Config, operation string, state string) string
 	return params
 }
 
-func EnsureState(state string) bool {
+func ensureState(state string) bool {
```

Comment:
Could you rename this function to something like: `validateState`?

Reply:
Updated, [0712bb6](https://github.com/hazelcast/hazelcast-commandline-client/pull/143/commits/0712bb6234fb499a008c1056f9c43b55a3e2754e)